sdk/python/tests/compiler/testdata/condition.py

```diff
@@ -39,8 +39,8 @@ def __init__(self, name, msg):
     
 
 @dsl.pipeline(
-  name='pipeline flip coin',
-  description='shows how to use dsl.Condition.'
+  name='Flip Coin Example Pipeline',
+  description='Shows how to use dsl.Condition().'
```

Comment:
auto-completion? ... or did you intentionally add the parentheses after `dsl.Condition`

Reply:
Not sure how this happened